client/my-sites/stats/controller.jsx

```diff
@@ -301,11 +301,10 @@ export function summary( context, next ) {
 	const extraProps =
 		context.params.module === 'videodetails' ? { postId: parseInt( queryOptions.post, 10 ) } : {};
 
-	let statsQueryOptions = {};
+	// The option is used for stats queries only.
+	const statsQueryOptions = pick( queryOptions, [ 'num', 'summarize', 'geoMode' ] );
```

Comment:
```suggestion
	const statsQueryOptions = pick( queryOptions, [ 'num', 'summarize' ] );
```
Do we need to get the parameter `geoMode` in this shared place for the Locations module dedicated argument?

Reply:
yes otherwise the parameter would not be passed to the summary component. I think this dtatsQueryOptions is meant for Api requests, so it filters out other parameters.